src/axom/primal/operators/detail/clip_impl.hpp

```diff
@@ -400,85 +400,95 @@ AXOM_HOST_DEVICE void poly_clip_reindex(Polyhedron<T, NDIMS>& poly,
 }
 
 /*!
- * \brief Finds the clipped intersection Polyhedron between Polyhedron
- *        poly and an array of Planes.
- *
- * \param [in] poly The polyhedron
- * \param [in] planes The array of planes
- * \param [in] eps The tolerance for plane point orientation.
+ * \brief Finds the polyhedron formed from clipping a polyhedron with a plane
  *
- * \return The Polyhedron formed from clipping the polyhedron with a set of planes.
+ * \param [in] poly The polyhedron to clip
+ * \param [in] plane The plane used to clip the polyhedron
+ * \param [in] eps The tolerance for plane point orientation
  *
+ * \return The polyhedron formed from clipping the input polyhedron with a plane
  */
 template <typename T, int NDIMS>
 AXOM_HOST_DEVICE Polyhedron<T, NDIMS> clipPolyhedron(
   Polyhedron<T, NDIMS>& poly,
-  axom::ArrayView<Plane<T, NDIMS>> planes,
+  const Plane<T, NDIMS>& plane,
   double eps)
 {
-  using PointType = Point<T, NDIMS>;
   using BoxType = BoundingBox<T, NDIMS>;
-  using PlaneType = Plane<T, NDIMS>;
 
-  //Bounding Box of Polyhedron
-  BoxType polyBox(&poly[0], poly.numVertices());
-
-  //Clip Polyhedron by each plane
-  for(PlaneType plane : planes)
+  // Check that plane intersects Polyhedron
+  if (intersect(plane, BoxType(&poly[0], poly.numVertices()), true, eps))
```

Comment:
Good to put a note that this `clipPolyhedron` overload functions slightly different from the others, that it only returns a Polyhedron if the polyhedron and plane intersect, else an empty Polyhedron.

Reply:
If they don't intersect, but the polyhedron is above the plane, it will return the original polyhedron, so this is the normal clip operation I'll do an intersection method in a different merge request.